frontend/src/components/ItemCreator.js

```diff
@@ -4,6 +4,13 @@ import { Box, Button, Form, FormField, Heading, TextInput } from "grommet";
 import React, { useState } from "react";
 import configData from "../config.json";
 
+const TYPE_NAME_TO_DB_TYPE_NAME = {
```

Comment:
Is this something that could be done using the config.json?

Reply:
Ah yes, started doing that but forgot to finish the job. Now done.